commands/create.go

```diff
@@ -109,8 +109,18 @@ func create(command *Command, args *Args) {
 	project := github.NewProject(owner, newRepoName, host.Host)
 	gh := github.NewClient(project.Host)
 
-	if gh.IsRepositoryExist(project) {
-		ui.Errorln("Existing repository detected. Updating git remote")
+	repo, err := gh.Repository(project)
+	if err != nil {
+		utils.Check(err)
+	}
+
+	if repo != nil {
```

Comment:
This can be `err == nil`. In that case you can be sure that `repo` is non-nil. No other checks are necessary